pkg/nsx/services/securitypolicy/wrap_test.go

```diff
@@ -163,18 +163,18 @@ func TestSecurityPolicyService_wrapResourceReference(t *testing.T) {
 	type args struct {
 		children []*data.StructValue
 	}
-	var children []*data.StructValue
-	serviceEntry := data.NewStructValue(
-		"",
-		map[string]data.DataValue{
-			"l4_protocol":   data.NewStringValue("TCP"),
-			"resource_type": data.NewStringValue("L4PortSetServiceEntry"),
-			// adding the following default values to make it easy when compare the existing object from store and the new built object
-			"marked_for_delete": data.NewBooleanValue(false),
-			"overridden":        data.NewBooleanValue(false),
-		},
-	)
-	children = append(children, serviceEntry)
+	// var children []*data.StructValue
```

Comment:
why we comment out this part?

Reply:
children is not set in tt.args.children, then the "children" variable is not used, which detected by golangci-lint tool.  Could you help check if this is needed? if not, I will delete it.

Reply:
I see. I checked the code, the part can be removed. Thanks